src/content/docs/synthetics/synthetic-monitoring/using-monitors/recheck-failed-monitors.mdx

```diff
@@ -44,7 +44,7 @@ To view a failed monitor's results and recheck it:
 />
 
 <figcaption>
-  **[one.newrelic.com**](https://one.newrelic.com) > Synthetics > (select a monitor) > Failures > (select a check failure):** To view details about a failed monitor check, click <Icon name="fe-rotate-cw"/>
+  **[one.newrelic.com](https://one.newrelic.com) > Synthetics > (select a monitor) > Failures > (select a check failure):** To view details about a failed monitor check, click <Icon name="fe-rotate-cw"/>
```

Comment:
I am not sure what "Run check" means here. Something to click on? We can fix this later.